### DISCUSSION TITLE
Great work! and what about local graphs?

### DISCUSSION THREAD

**Discussion by mutjan** [natefrisch01/Graph-Link-Types#21] [General]
Thanks for this great idea.
But I thought it would be even more useful in local graphs.
It's hard to check the link types when the globle graph has too many notes.
But local graph will always be a good place to dive in the relationship of notes.
Hope to see this work in local graphs in future updates

**Comment by natefrisch01** [comment]
![image](https://github.com/natefrisch01/Graph-Link-Types/assets/44580969/5e8e8bb2-cd89-4cd0-aae9-99d69aa2fd2b)

**Reply by natefrisch01** [reply]
It should work for local graphs... perhaps there's something else causing it to not work for you? How are your links formatted, what are your filenames, and are they in subfolders? Perhaps this is causing it to not work for local graph.

**Comment by mutjan** [comment]
Thanks for the help.
I create a brand new vault and it works, now I'll try to find what cause this problem

**Reply by mutjan** [reply]
finally nailed it.

if there are empty properties, then this plugin only render properties as link type, ignoring those defined inline with dataview syntax

<img width="1318" alt="image" src="https://github.com/natefrisch01/Graph-Link-Types/assets/17292215/f64c6080-75e3-48f5-a17c-c173aec0afa1">

fill the empty property and refresh, back to normal

<img width="1327" alt="image" src="https://github.com/natefrisch01/Graph-Link-Types/assets/17292215/65182413-bc6c-4c14-a5e3-bcff7ee14f32">

**Reply by mutjan** [reply]
furture test shows empty inline link also cause the issue
more accurately, it stops at the last filled link
<img width="944" alt="image" src="https://github.com/natefrisch01/Graph-Link-Types/assets/17292215/8d4a9f1d-dd5f-4b87-8f1d-11eeea08d907">

**Reply by mutjan** [reply]
#31 added a simple fix
